src/main/webapp/app/assessment-instructions/assessment-instructions.component.html

```diff
@@ -10,9 +10,7 @@ <h3 class="card-title">
         </div>
         <div class="card-body markdown-preview">
             <h3>{{ exercise.title }}</h3>
-            <br />
-            <jhi-expandable-paragraph header="{{ 'assessmentInstructions.problemStatement' | translate }}" [text]="formattedProblemStatement"></jhi-expandable-paragraph>
-            <br />
+            <jhi-expandable-problem-statement [exercise]="exercise"></jhi-expandable-problem-statement>
             <jhi-expandable-sample-solution [exercise]="exercise"></jhi-expandable-sample-solution>
             <br />
```

Comment:
I think we should either have `<br />` between all sections or remove them completely.